helper-cli/src/main/kotlin/commands/ImportCopyrightGarbageCommand.kt

```diff
@@ -65,8 +65,7 @@ internal class ImportCopyrightGarbageCommand : CliktCommand(
             emptySet<String>()
         }
 
-        val locale = Locale.Builder().setLanguage("en").setRegion("US.utf-8").setVariant("POSIX").build()
-        val collator = Collator.getInstance(locale)
+        val collator = Collator.getInstance(Locale("en", "US.utf-8", "POSIX"))
```

Comment:
Looks like this could instead be fixed by changing `"US.utf-8"` to `"US"` in the current code. Not sure what the encoding has to do with the locale anyway; the `Locale` constructor seems to have been tolerant about this. Does stripping the ".utf-8" part work for you, too?

Reply:
Thanks. That works for me too. I've updated the commit accordingly!